chart/fluxcloud/templates/secret.yaml

```diff
@@ -0,0 +1,26 @@
+{{- with .Values.config -}}
+{{- if or .slack.hook.url .datadog.api.key .datadog.app.key .msTeams.hook.url .webhook.url -}}
+apiVersion: v1
+data:
+  {{- if .slack.hook.url }}
+  {{ default "slack-webhook" .slack.hook.secret.key }}: {{ .slack.hook.url | b64enc | quote }}
+  {{- end }}
+  {{- if .datadog.api.key }}
```

Comment:
Wouldn't this break when the key datadog.api doesn't exists?

Reply:
`.datadog.api.key` and all the other keys/urls are defined as empty string in the default values file so it won't break, instead it will evaluate the condition to false.

Reply:
Got it, thanks 👍